### PR TITLE
Add kernel regression test memeat and memtester

### PR DESCRIPTION
**Pull request by SeroSun**
I add two stable testcases for kernel regression test. memeat need around 10min, and memtester need around half hour to run. Both of them are tested in local server in Beijing successfully. I suggest to add these two test into openqa.suse.de test envirorment for new build test.

### CODE REVIEW

**Comment by coolo** [comment]
Try this:
```
ln -s ../os-autoinst/tools .
./tools/tidy

git commit --amend -a

File: tests/qa_automation/kernel_memeat.pm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+use base "qa_run";
```
Comment (okurz) [inline tests/qa_automation/kernel_memeat.pm]: Add copyright header

File: tests/qa_automation/kernel_memtester.pm [hunk 2]
```diff
@@ -0,0 +1,16 @@
+use base "qa_run";
```
Comment (okurz) [inline tests/qa_automation/kernel_memtester.pm]: add copyright header

**Comment by SeroSun** [comment]
I don't know why after I used this command line "git commit --amend -a", there create a merge here...Actually I didn't change anything after I add copyright.

**Comment by coolo** [comment]
because your branch was heavily outdated and missed that master already had copyright for these files. Let me fix this - and you make sure your fork is uptodate with the upstream repo.

**Comment by coolo** [comment]
btw, it's 2016 :)

**Comment by coolo** [comment]
Obsoleted by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1053

**Comment by SeroSun** [comment]
Thank you coolo :)
I'll update my fork.

**Pull request closed without merging**